build.fsx

```diff
@@ -227,12 +225,23 @@ Target "ReleaseBinaries" <| fun () ->
     createRelease() 
     publishFiles "binaries" "release" "bin" "bin" 
 
+Target "TestSourcelink" <| fun () ->
+    let testSourcelink framework proj =
+        let basePath = Path.GetFileNameWithoutExtension proj
+        let pdb = sprintf "bin/Release/netstandard2.0/%s.pdb" basePath
+        DotNetCli.RunCommand (fun p -> { p with ToolPath = getSdkPath(); WorkingDir = Path.GetDirectoryName proj }) (sprintf "sourcelink test %s" pdb)
+
+    ["net45"; "netstandard2.0"]
+    |> Seq.collect (fun fw -> buildProjs |> Seq.map (testSourcelink fw))
+    |> Seq.iter id
+
 Target "Release" DoNothing
 
 "CleanDocs" ==> "GenerateDocs" ==> "ReleaseDocs"
 "ReleaseDocs" ==> "Release"
 "ReleaseBinaries" ==> "Release"
 "NuGet" ==> "Release"
+"TestSourcelink" ==> "Release"
```

Comment:
Update help bellow to mention this new target

Reply:
Thanks for the reminder!